tests/markers.py

```diff
@@ -13,6 +13,8 @@
 parametrize = mark.parametrize
 test_case = pytestrail.case
 xfail = mark.xfail
+skip = mark.skip
+otto = mark.otto
```

Comment:
Remove otto marker. If you need a marker as your working then add it on the fly but we shouldn't have user specific info with the framework. A person will come along later and ask, "What does the otto marker mean?" and no one will know.

Reply:
@m1yag1 marker removed. pushed changes. the circleci check came up with 2 fails in webview tests. they are completely unrelated and passed previously.

Reply:
@m1yag1 ``` assert False  +  where False = <pages.webview.base.Page.Footer object at 0x7f133da83be0>.is_android_app_link_displayed ```

android app link has been removed on staging. that is the fail

Reply:
@m1yag1 
1. I removed android app link from test_home in another PR - this is OK
2. the other fail is related to the fact that the book redirects:
```
Failed for this reason:

test_in_book_search[e42bd376-624b-4c0f-972f-e0c57998e765-mitosis genetics-True-0-True-False] - tests.webview.ui.test_content

tests/webview/ui/test_content.py
webview_base_url = 'https://staging.cnx.org'
```
Book e42bd376-624b-4c0f-972f-e0c57998e765 redirects. Not related to this change - OK